bc_obps/bc_obps/settings.py

```diff
@@ -35,6 +35,12 @@
 ENVIRONMENT = os.environ.get("ENVIRONMENT")
 BYPASS_ROLE_ASSIGNMENT = os.environ.get("BYPASS_ROLE_ASSIGNMENT", False) == "True"
 
+# CHES-specific variables
+CHES_CLIENT_ID = os.environ.get("CLIENT_ID")
+CHES_CLIENT_SECRET = os.environ.get("CLIENT_SECRET")
```

Comment:
I suggest using the same names (`CHES_CLIENT_ID`, `CHES_CLIENT_SECRET` and `CHES_TOKEN_ENDPOINT`) as we might have other similar variables, which could be confusing. (This applies to other files too)